test/integration/kubernetes.test.ts

```diff
@@ -314,3 +368,19 @@ tap.test('notify upstream of deleted pods that have no OwnerReference', async (t
     'snyk-monitor sends deleted workloads to upstream for pods without OwnerReference',
   );
 });
+
+tap.test('snyk-monitor secure configuration is as expected', async (t) => {
+  const kubeConfig = new KubeConfig();
+  kubeConfig.loadFromDefault();
+  const k8sApi = kubeConfig.makeApiClient(AppsV1Api);
+
+  const response = await k8sApi.readNamespacedDeployment(
+    'snyk-monitor',
+    'snyk-monitor',
+  );
+  const deployment = response.body;
+
+  validateSecureConfiguration(t, deployment);
+  validateVolumeMounts(t, deployment);
+  validateEnvironmentVariables(t, deployment);
+});
```

Comment:
Is this accidentally copied from somewhere?

Reply:
😱

Reply:
I see this is resolved but the code is still here, why do we check the secure configuration? Isn't this tested similarly in a different place?

Reply:
no, it shouldn't be here at all, sorry 🤦‍♀️